controller/scheduler/swarm.py

```diff
@@ -6,7 +6,6 @@
 
 from .states import JobState
 
-
```

Comment:
test-style uses flake8. I ran flake8 manually and didn't throw any errors for swarm.py.